plot_Draft  Teil

LJinzhen opened the pull request:
Ich habe ein paar Diagramme gezeichnet, damit wir sie diskutieren können.

LJinzhen committed:
Create Schritt.3_Li.R

yiyi808 committed:
Merge pull request #16 from yiyi808/main

Daten

LJinzhen committed:
Create Teil4.R

LJinzhen committed:
T4_a

LJinzhen committed:
Update README.md

LJinzhen committed:
Delete T4_a.R

LJinzhen committed:
Rename Teil4.R to Teil4_1.R

LJinzhen committed:
Create plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Update plot_Draft1.R

Xinyi-Jing commented:
Ich finde die Grafik hübsch, ich habe nur die untere Achse und den Namen geändert

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen commented:
Ich habe zwei weitere Diagramme gezeichnet. Die Histogramme für die Teile a) & b) unseres Plans sind jetzt gezeichnet, und wenn es keine Probleme gibt, werde ich anfangen, die Diagramme zu verschönern.

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Update plot_Draft1.R

Ich habe einige Erläuterungen hinzugefügt und Änderungen am Inhalt Ihrer Datei vorgenommen. Wir können sie besprechen und erneut optimieren.

Xinyi-Jing commented:
Ich habe einige Erläuterungen hinzugefügt und Änderungen am Inhalt Ihrer Datei vorgenommen. Wir können sie besprechen und erneut optimieren.

Xinyi-Jing committed:
Update plot_Draft1.R

Xinyi-Jing commented:
Ich denke, es wäre besser, wenn wir ein Diagramm des Anteils der Menschen mit hohen und niedrigen Niveaus in Mathematik oder Programmieren in jedem Studienfach zeichnen könnten. Das heißt, wenn ein Studienfach eine Spalte ist, wird die Spalte in drei Teile unterteilt: hoch, mittel und niedrig. Der hohe Anteil repräsentiert die Anzahl der Personen mit hohem Interesse , der mittlere und der niedrige Anteil usw. Auf diese Weise können wir deutlich erkennen, dass Menschen in verschiedenen Studienfach unterschiedliches Interesse an Mathematik oder Programmierung haben.

Xinyi-Jing committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Update plot_Draft1.R

Ich habe alle Analysen durchgeführt. Alle unsere Diagramme und Analysen sind integriert, ich habe Ihre Diagrammdatei bearbeitet. Du kannst es dir ansehen. Wenn es kein Problem gibt, können wir es gemeinsam in den Teil 4-Branch einfügen.

Xinyi-Jing commented:
Ich habe alle Analysen durchgeführt. Alle unsere Diagramme und Analysen sind integriert, ich habe Ihre Diagrammdatei bearbeitet. Du kannst es dir ansehen. Wenn es kein Problem gibt, können wir es gemeinsam in den Teil 4-Branch einfügen.

Xinyi-Jing committed:
Update plot_Draft1.R

LJinzhen committed:
Update plot_Draft1.R

LJinzhen commented:
Ich habe es gesehen und meinen Teil davon hochgeladen, aber es gibt noch einen Teil, der diskutiert werden muss

Xinyi-Jing commented:
OK.Wir prüfen das alle noch einmal.

LJinzhen committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Update plot_Draft1.R

Xinyi-Jing committed:
Create plot_Draft1.R

Xinyi-Jing commented:
Ich denke, es gibt kein Problem. Wir können es in Teil 4 einfügen.

LJinzhen commented:
Ok
